examples/bottlerocket-construct/index.ts

```diff
@@ -7,13 +7,14 @@ import * as ssp from '../../lib'
 import * as team from '../teams'
 
 export default class BottlerocketConstruct extends cdk.Construct {
-    constructor(scope: cdk.Construct, id: string) {
-        super(scope, id);
+    constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+        super(scope, id)
 
         // Setup platform team
-        const accountID = process.env.CDK_DEFAULT_ACCOUNT!
-        const platformTeam = new team.TeamPlatform(accountID)
-        const teams: Array<ssp.Team> = [platformTeam];
+        const accountID = props?.env?.account
```

Comment:
You can use `props?.env?.account!` and drop `<string>` in arguments, however the question is what is going to happen if account is not passed? I would expect fail-fast and explicit assertion.

Reply:
Lint is currently setup to not like not null assumptions.   I've not fixed made this change.   
`  const accountId = props?.env?.account!`
I prefer the <string>  Let me know.

Reply:
I like explicit tbh coming from mostly C/C++/Java background :). My main objective was consistency throughout the project. We can keep your style. For assertions of input parameters, you could leverage `console. assert` as opposed to explicit throw.